dt-metrics/metrics-workers.php

```diff
@@ -532,19 +532,19 @@ public function get_workers_data( $force_refresh = false ) {
         $times = $wpdb->get_results( "
             SELECT AVG(time) as average_time_to_attempt, users.ID
             from (
-                SELECT contacts.ID, 
-                    MIN(date_assigned.hist_time) as date_assigned, 
-                    MIN(date_attempted.hist_time) as date_attempted, 
+                SELECT contacts.ID,
+                    MIN(date_assigned.hist_time) as date_assigned,
+                    MIN(date_attempted.hist_time) as date_attempted,
                     MIN(date_attempted.hist_time) - MIN(date_assigned.hist_time) as time,
-                    pm.meta_value as user 
+                    pm.meta_value as user
                 from $wpdb->posts as contacts
                 INNER JOIN $wpdb->postmeta as pm on ( contacts.ID = pm.post_id AND pm.meta_key = 'assigned_to' )
                 INNER JOIN $wpdb->dt_activity_log as date_attempted on ( date_attempted.meta_key = 'seeker_path' and date_attempted.object_type = 'contacts' AND date_attempted.object_id = contacts.ID AND date_attempted.meta_value != 'none' )
                 INNER JOIN $wpdb->dt_activity_log as date_assigned on ( date_assigned.meta_key = 'overall_status' and date_assigned.object_type = 'contacts' AND date_assigned.object_id = contacts.ID AND date_assigned.meta_value = 'assigned' )
                 WHERE date_attempted.hist_time > date_assigned.hist_time
                 GROUP by contacts.ID
             ) as times
-            LEFT JOIN $wpdb->users as users on ( times.user = CONCAT( 'user-', users.ID ) ) 
+            LEFT JOIN $wpdb->users as users on ( times.user = CONCAT( 'user-', users.ID ) )
```

Comment:
@corsacca do you want me to put back these spaces and the end of the lines that my coding app automatically removed? (there are heaps of them)

Reply:
just don't commit them next time.